metadata-db-app/src/cmr/metadata_db/services/concept_constraints.clj

```diff
@@ -160,6 +160,7 @@
    :service [(partial pfn-constraint :service-name)]
    :tool [(partial pfn-constraint :tool-name)]
    :subscription [(partial pfn-constraint :subscription-name)]
+   ;; will remove this fingerprint check when updating tests later.
```

Comment:
is there a standard or practice to use "TODO" or something like that to make it easy to find these latter with grep?

Reply:
No. just decided not to do it for this ticket since it might break some tests.